addon/templates/components/polaris-radio-button.hbs

```diff
@@ -1,27 +1,27 @@
-{{#polaris-choice
-  inputId=_id
-  label=label
-  labelHidden=labelHidden
-  disabled=disabled
-  helpText=helpText
-}}
-  <span data-test-radio-button class="Polaris-RadioButton">
+<PolarisChoice
+  @inputId={{this._id}}
+  @label={{@label}}
+  @labelHidden={{@labelHidden}}
+  @disabled={{@disabled}}
+  @helpText={{@helpText}}
+>
+  <span data-test-radio-button="true" class="Polaris-RadioButton">
     <input
       data-test-radio-button-input
       type="radio"
       class="Polaris-RadioButton__Input"
-      id={{_id}}
-      name={{name}}
-      value={{value}}
-      checked={{checked}}
-      disabled={{disabled}}
-      aria-describedby={{describedBy}}
-      onchange={{action onChange value="target.value"}}
-      onfocus={{action onFocus}}
-      onblur={{action onBlur}}
+      id={{this._id}}
+      name={{@name}}
+      value={{@value}}
+      checked={{@checked}}
+      disabled={{@disabled}}
+      aria-describedby={{this.describedBy}}
+      {{on "change" (or @onChange this.onChange)}}
```

Comment:
I think we still want to pass the value to the `onChange` action....assuming we were correct and that's what the react implementation does
PS: we shouldn't modify tests to match our implementation.....but make sure tests are testing correct behavior and we pass 😬

Reply:
check the [example](https://github.com/smile-io/ember-polaris/pull/504#discussion_r380267497) I've posted earlier....that should be in the component not in the tests
let me know if this doesn't makes sense pls

Reply:
Ah so you mean we want a middle-man method (something like a private `handleChange` action) where we grab the value and _then_ call `onChange`?

Originally when you said 

> we can just handle in the local action

I thought you meant that the passed-in `onChange` would be responsible for this. Sorry about the confusion 😬

Reply:
yeah, like `handleChange`
moving this to the caller `onChange` would mean we break the API of this component...which we don't want